ingest-app/src/migrations/006_update_bulk_update_tables.clj

```diff
@@ -0,0 +1,87 @@
+(ns migrations.006-update-bulk-update-tables
+  (:require [clojure.java.jdbc :as j]
+            [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 6."
+  []
+  (println "migrations.006-update-bulk-update-tables up...")
+  (j/with-db-transaction [trans-conn (config/db)]
+    (j/db-do-commands trans-conn "LOCK TABLE CMR_INGEST.bulk_update_task_status IN EXCLUSIVE MODE")
+    (j/db-do-commands trans-conn "LOCK TABLE CMR_INGEST.bulk_update_coll_status IN EXCLUSIVE MODE")
+    (j/db-do-commands trans-conn "CREATE TABLE CMR_INGEST.bulk_update_task_status_new (
+                                  TASK_ID VARCHAR(255) NOT NULL,
+                                  PROVIDER_ID  VARCHAR(10) NOT NULL,
+                                  REQUEST_JSON_BODY  BLOB NOT NULL,
+                                  STATUS VARCHAR(20),
+                                  STATUS_MESSAGE   VARCHAR(255),
+                                  CREATED_AT TIMESTAMP(9) WITH TIME ZONE DEFAULT sysdate NOT NULL)")
+    (j/db-do-commands trans-conn "CREATE TABLE CMR_INGEST.bulk_update_coll_status_new (
+                                  TASK_ID VARCHAR(255) NOT NULL,
+                                  CONCEPT_ID VARCHAR(255) NOT NULL,
+                                  STATUS VARCHAR(20),
+                                  STATUS_MESSAGE  VARCHAR(500),
+                                  ACTION_TAKEN VARCHAR2(1) DEFAULT 'F' NOT NULL)")
```

Comment:
Chris, I was researching on Boolean in oracle, and saw varchar2(1) was recommended to replace boolean since it doesn't exist.  it's only gonna be false and true, which will be "F" and "T".  I guess char will be fine.  Thanks. Done.